server/controllers/conversationController.js

```diff
@@ -26,50 +26,10 @@ exports.create = async (req, res, next) => {
 };
 
 // grabs all conversations from auth user, populates the users, and grabs the most recent message and populates the author
-
 exports.read = async (req, res, next) => {
   try {
-    const data = await Conversation.aggregate([
-      { $match: { users: mongoose.Types.ObjectId(req.user._id) } },
-      {
-        $lookup: {
-          from: "users",
-          localField: "users",
-          foreignField: "_id",
-          as: "users",
-        },
-      },
-      {
-        $lookup: {
-          from: "messages",
-          let: { id: "$_id" },
-          pipeline: [
-            {
-              $match: {
-                $expr: { $eq: ["$$id", "$conversation"] },
-              },
-            },
-            { $sort: { dateCreated: -1 } },
-            { $limit: 1 },
-            {
-              $lookup: {
-                from: "users",
-                let: { id: "$author" },
-                pipeline: [
-                  {
-                    $match: {
-                      $expr: { $eq: ["$$id", "$_id"] },
-                    },
-                  },
-                ],
-                as: "author",
-              },
-            },
-          ],
-          as: "messages",
-        },
-      },
-    ]);
+    console.log(convoQuery(req.user._id));
```

Comment:
Make sure to get rid of this console.log.

Reply:
Done!